helm/hpcc/values.schema.json

```diff
@@ -929,6 +929,16 @@
       },
       "additionalProperties": false
     },
+    "thorSubComponent": {
+      "description": "Thor sub-component with resource and cost configuration",
```

Comment:
currently only cost configuration, not resources? But (see other comment) it would be good to standarize on this schema, i.e. move "resources" under the subcomponent (but in separate JIRA).

Reply:
To keep the PR small and simply, I have limited the changes in the PR to adding the cost definitions only. The other changes as suggested about is planned in a separate PR (I'll be adding additional items under the parent ticket: https://hpccsystems.atlassian.net/browse/HPCC-34436)